examples/src/python/example/python_distribution/hello/fasthello/setup.py

```diff
@@ -8,8 +8,8 @@
 from distutils.core import Extension
 
 
-c_module = Extension(str('c_greet'), sources=[str('c_greet.c')])
-cpp_module = Extension(str('cpp_greet'), sources=[str('cpp_greet.cpp')])
+c_module = Extension(b'c_greet', sources=[b'c_greet.c'])
```

Comment:
I'm assuming the call to `str()` around string literals was to convert into byte strings, because `from __future__ import unicode_literals` is toggled on? If not, and unicode works, we can remove both `b` prefix and `str()`.